pages/advanced/cli-demos.mdx

```diff
@@ -0,0 +1,15 @@
+# Demos
+
+## Create a new Safe
+
+The `PRIVATE_KEY` environment variable for this demo was set to an _EOA_ private key.
+
+[![Create a new Safe](https://asciinema.org/a/0jdHGLVRrkS9URxPoZ8ZJ7W2C.svg)](https://asciinema.org/a/0jdHGLVRrkS9URxPoZ8ZJ7W2C)
+
+## Sending Ether using a Trezor hardware wallet
+
+[![Sending Ether using a Trezor hardware wallet](https://asciinema.org/a/9BrQKYQRXbysmEL8rw1jDWZhn.svg)](https://asciinema.org/a/9BrQKYQRXbysmEL8rw1jDWZhn)
+
+## Create a remove owner transaction in the Transaction Service
```

Comment:
This sounds artificial. Maybe: "Create a transaction in the Safe Transaction Service to remove an owner". Same in the link